### PR TITLE
Fix incomplete HeaderRW max checking

### PR DESCRIPTION
**Pull request by jcorbin**
Follow on to #743

### CODE REVIEW

File: node/errors.js [hunk 1]
```diff
@@ -442,8 +442,9 @@ module.exports.ThriftHeadStringifyError = WrappedError({
 
 module.exports.TooManyHeaders = TypedError({
     type: 'tchannel.protocol.too-many-headers',
-    message: 'Got too many headers. Expected at most 128 but got: {count}',
+    message: 'too many transport headers, got {count}, expected at most {maxHeaderCount}',
```
Comment (Raynos) [inline node/errors.js]: I hardcoded the message in the test. Test needs updating

**Comment by Raynos** [comment]
Lgtm

**Comment by jcorbin** [comment]
yup, and by new byteLength branches should've introduce coverage exceptions... over to you @Raynos

**Comment by Raynos** [comment]
:D thanks @jcorbin

**Comment by Raynos** [comment]
this lgtm ;)

**Pull request merged**